internal/pkg/poster/poster.go

```diff
@@ -160,13 +165,18 @@ func (p *Poster) sendPoSt(ctx context.Context, stateView *appstate.View, candida
 		return err
 	}
 
-	_, _, err = p.outbox.Send(
+	signerAddr, err := stateView.AccountSignerAddress(ctx, workerAddr)
+	if err != nil {
+		return err
+	}
+
+	mcid, _, err := p.outbox.Send(
 		ctx,
-		workerAddr,
+		signerAddr,
```

Comment:
To save onchain space / gas we may want to use the workerAddr but give the outbox the capability to lookup the corresponding signing address and key.

Reply:
Thought of that. This is a bit of a quick fix. The Outbox is already accessing actor state, so it's not much of an architectural shift to have it look up signing addresses. I'll take a stab at it.

Reply:
Fine to file an issue for it and come back later.

Reply:
#3900